src/node/rpc/node_frontend.h

```diff
@@ -139,19 +139,18 @@ namespace ccf
       {
         case QuoteVerificationResult::Failed:
           return std::make_pair(
-            HTTP_STATUS_INTERNAL_SERVER_ERROR, "Quote could not be verified");
+            HTTP_STATUS_UNAUTHORIZED, "Quote could not be verified");
```

Comment:
Since the quote is supplied by the client on join, the error status when the quote is invalid should be client rather than server errors.

Reply:
Not sure if this is true of all these errors. If the local verification toolchain blows up, that's still a `SERVER_ERROR`? But I agree for the known failure codes.

Reply:
Good point. I'll change the generic error to a `SERVER_ERROR`.